components/core/src/clp_s/search/ast/DateLiteral.cpp

```
@@ -2,77 +2,55 @@
+bool DateLiteral::as_int(int64_t& ret, FilterOperation op) {
+    ret = m_default_precision_timestamp;
     return true;
 }
 
-bool DateLiteral::as_var_string(std::string& ret, FilterOperation op) {
-    if (op == FilterOperation::LT || op == FilterOperation::GT || op == FilterOperation::LTE
-        || op == FilterOperation::GTE)
-    {
-        return false;
-    }
+bool DateLiteral::as_float(double& ret, FilterOperation op) {
```

Cppcheck (2.18.0)

[style] The function 'as_float' is never used.

(unusedFunction)